roles/openmrs/tasks/main.yml

```diff
@@ -43,18 +43,18 @@
   become: yes
   become_user: root
   file:
-    path: "/usr/share/tomcat7/.OpenMRS"
-    owner: "{{ tomcat_user }}"
+    path: "{{openmrs_home_directory}}"
+    owner: "{{ openmrs_system_user }}"
     group: "{{ tomcat_group }}"
     state: directory
-    mode: 0755
+    mode: g+rw
```

Comment:
The mode is explicit about the capabilities for the group, but not the owner and "other".

Reply:
For this directory we need a read/ wright for for openSRP & openMRS

Reply:
Cool. Would `0775` be a nice octal then? It's explicit about the capabilities for all types of users (not just the group).